docs/network/evm-nodes.md

```diff
@@ -7,38 +7,65 @@ The Etherlink EVM nodes are responsible for maintaining a copy of the Etherlink
-See [Installing Octez](https://tezos.gitlab.io/introduction/howtoget.html).
-1. Set the `sr_node_observer_rpc` environment variable to the URL to the Smart Rollup node, such as `http://localhost:8932`.
+Besides, we have also published a [minimal Docker image](https://hub.docker.com/layers/tezos/tezos-bare/octez-evm-node-v0.1/images/sha256-e5fdf87f2827dd971e2097faff88c6c6ce53793a2191a903a6a905a647c051f5?context=explore) containing the daemon.
+
+## Initializing the data directory
+
+1. If you want your EVM node to check the correctness of the blueprints it receives via a Smart Rollup node, set the `sr_node_observer_rpc` environment variable, such as `http://localhost:8932`.
 1. Set the `evm_observer_dir` environment variable to the directory where the node should store its local data.
 The default is `$HOME/.octez-evm-node`.
-1. Initialize the node by running this command:
+1. Initialize the node. To trust incoming blueprints, use `--dont-track-rollup-node`:
+
+   ```bash
+   octez-evm-node init config \
+     --data-dir $evm_observer_dir --dont-track-rollup-node \
+     --preimages-endpoint https://snapshots.eu.tzinit.org/etherlink-mainnet/wasm_2_0_0 \
+     --evm-node-endpoint https://relay.mainnet.etherlink.com
+   ```
+
+   Alternatively, if you want to rely on a Smart Rollup node to check the correctness of blueprints coming from the sequencer, use `--rollup-node-endpoint`:
 
    ```bash
-   octez-evm-node init config --devmode \
+   octez-evm-node init config \
      --data-dir $evm_observer_dir --rollup-node-endpoint $sr_node_observer_rpc \
      --preimages-endpoint https://snapshots.eu.tzinit.org/etherlink-mainnet/wasm_2_0_0 \
-     --evm-node-endpoint https://node.mainnet.etherlink.com
+     --evm-node-endpoint https://relay.mainnet.etherlink.com
    ```
 
    This configuration uses the preimages that the Tezos Foundation hosts on a file server on a so-called "preimages endpoint".
    It's safe to use these preimages because the node verifies them.
    If you don't want to use third-party preimages, you can build the kernel yourself and move the contents of the `wasm_2_0_0/` directory to the local data directory; see [Building the Etherlink kernel](/network/building-kernel).
    However, in this case, you must manually update this directory with the preimages of every kernel voted by the community and deployed on Etherlink after that.
 
-1. Set the `sr_observer_data_dir` environment variable to the location of the data directory for the Smart Rollup node.
-The default value is `$HOME/.tezos-smart-rollup-node`.
+## Running the node
+
+### From an existing Etherlink Smart Rollup node
+
+1. Download [a Smart Rollup node snapshot](https://snapshots.eu.tzinit.org/etherlink-ghostnet/), and use the `octez-smart-rollup-node` executable to import it in a temporary directory.
```

Comment:
```suggestion
1. Download [an Etherlink Smart Rollup node snapshot](https://snapshots.eu.tzinit.org/etherlink-ghostnet/), and use the `octez-smart-rollup-node` binary to import it in a temporary directory.
```

This is a link to Ghostnet but the command uses Mainnet.

Reply:
:+1: